include/vrv/functorparams.h

```diff
@@ -1768,6 +1768,28 @@ class SetChildZonesParams : public FunctorParams {
     Doc *m_doc;
 };
 
+//----------------------------------------------------------------------------
+// FindNextOfTypeParams
+//----------------------------------------------------------------------------
+
+/**
+ * member 0: the attComparison text
```

Comment:
Argument takes `Comparison` not `AttComparison` and text does not factor into this parameter.